pkg/parse/run.go

```diff
@@ -129,14 +127,13 @@ func Run(ctx context.Context, p Parser) {
 			// from an older commit.
 			if state.syncStatus.commit == state.sourceStatus.commit &&
 				state.syncStatus.commit == state.renderingStatus.commit {
+				// Combine remediator and applier errors
+				syncErrs := status.Append(p.RemediatorConflictErrors(), p.ApplierErrors())
+
 				klog.V(3).Info("Updating sync status (periodic while not syncing)")
-				if err := p.SetSyncStatus(ctx, p.ApplierErrors()); err != nil {
+				if err := p.SetSyncStatus(ctx, syncErrs); err != nil {
 					klog.Warningf("failed to update remediator errors: %v", err)
 				}
-				remediatorErrs := p.RemediatorConflictErrors()
-				if len(remediatorErrs) > 0 {
-					UpdateConflictManagerStatus(ctx, remediatorErrs, p.K8sClient())
-				}
```

Comment:
This is to report a similar conflict error to the other conflicting RootSync. Do we want to remove that?

Reply:
Reverted this removal and moved it into `setSyncStatus`.